### PR TITLE
Closes #6287: Update Not Yet Supported caption string

### LINKED ISSUE #6287: Update "Not yet available" copy
The following copy needs to be updated. @betsymi has the correct copy. 

Screenshot_20200313-143124_Firefox_Nightly

### PR DESCRIPTION
**Pull request by psymoon**
---

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/feature/addons/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -128,7 +128,7 @@
     <!-- The content of the notification, this will be shown to the user when more than two newly supported add-ons are available. %1$s is the app name (in most cases Firefox). -->
     <string name="mozac_feature_addons_supported_checker_notification_content_more_than_two">Add them to %1$s</string>
     <!-- This is the caption for not yet supported screen caption -->
-    <string name="mozac_feature_addons_not_yet_supported_caption">Firefox add-on technology is modernizing. These add-ons use frameworks that are not compatible with Firefox 75 &amp; beyond.</string>
+    <string name="mozac_feature_addons_not_yet_supported_caption">We\'re currently building support for an initial selection of Recommended Extensions.</string>
```
Comment (Amejia481) [inline components/feature/addons/src/main/res/values/strings.xml]: Unfortunately, I believe when we update a string we need to use a new string key. @flodolo or @Pike can help us to confirm.

Comment (flodolo) [reply components/feature/addons/src/main/res/values/strings.xml]: We do need a new ID. Also, please don't use escaped single quote, use an actual apostrophe `‘`.

Comment (Amejia481) [reply components/feature/addons/src/main/res/values/strings.xml]: @flodolo should we delete all the previous strings?

Comment (Amejia481) [inline components/feature/addons/src/main/res/values/strings.xml]: @psymoon you could use `mozac_feature_addons_not_yet_supported_caption2` :)

Comment (flodolo) [reply components/feature/addons/src/main/res/values/strings.xml]: Normally, yes, you should remove a string if it's unused. 

But I'm not exactly sure how this work here (@Pike ?). I know Fenix keeps them around until that specific strings it's in use (i.e. shipping), no clue about a-c

Comment (Pike) [reply components/feature/addons/src/main/res/values/strings.xml]: I don't know the state of release branches at this time.

**Comment by psymoon** [comment]
bors r=Amejia481

File: components/feature/addons/src/main/res/values/strings.xml (hunk 1 above)
Comment (flodolo) [reply components/feature/addons/src/main/res/values/strings.xml]: @pocmo or @st3fan 
What is the expectation on your side at this point? I wasn't involved in the original discussion, and I have no clue what's the current status.

Comment (pocmo) [reply components/feature/addons/src/main/res/values/strings.xml]: I see no problem in removing the old string as soon as it is no longer used.

**Pull request merged**